internal/tiltfile/docker.go

```diff
@@ -308,14 +308,14 @@ func (s *tiltfileState) mountsToDomain(image *dockerImage) []model.Mount {
 	return result
 }
 
-func (s *tiltfileState) reposToDomain(image *dockerImage) []model.LocalGitRepo {
+func pathsToRepos(paths []localPath) []model.LocalGitRepo {
```

Comment:
fwiw this func name confused me -- seems like we're converting the paths to repos? consider something like `reposForPaths` (ugh, or something better, don't love that either) and/or a docstring.

Reply:
yeah, don't love that either, but don't see a better option (and I wasn't a big fan of the existing name before this change, either)